recipes/maker/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
     - snap
     - snoscan
     - perl
-    - perl-bioperl >=1.7
+    - perl-bioperl-core >=1.007002=pl526_1
```

Comment:
Is it necessary to pin the build number too (pl526_1)? I would just write `>=1.007002` (here and on line 52)